.github/workflows/cron_jerigon_zero_testing.yml

```
@@ -0,0 +1,106 @@
+name: Jerigon Zero Testing
+
+on:
+  # TODO - Change this before merge
+  # # Uncomment when ready to run on a schedule
+  # schedule:
+  #   # Run every Sunday at 12:00 AM (UTC)
+  #   - cron: "0 12 * * SUN"
+  push:
+    branches: [develop]
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch:
+    branches:
+      - "**"
+
+env:
+  CARGO_TERM_COLOR: always
+  REGISTRY: ghcr.io
+
+jobs:
+  jerigon_zero_testing:
+    name: Jerigon Zero Testing - Integration and Benchmarking
+    runs-on: zero-reg
+    concurrency:
+      group: jerigon_zero_testing
+      cancel-in-progress: true
+    steps:
+      - name: Checkout zk_evm code
+        uses: actions/checkout@v4
+
+      - name: Setup Rust Toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+
+      - name: Set up Rust Cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+
+      - name: Build the project
+        run: |
+          RUSTFLAGS='-C target-cpu=native -Zlinker-features=-lld -Copt-level=3' cargo build --release
```

```suggestion
          RUSTFLAGS='-C target-cpu=native -Zlinker-features=-lld' cargo build --release
```

`--release` implies `opt-level=3`. I think @atanmarko wrote [the same thing](https://github.com/0xPolygonZero/zk_evm/pull/701#discussion_r1830865164) - are you both cribbing from the same source? Can you point me to it so I can fix it?